ibis/backends/tests/test_join.py

```diff
@@ -17,7 +17,9 @@ def _pandas_semi_join(left, right, on, **_):
 def _pandas_anti_join(left, right, on, **_):
     assert len(on) == 1, str(on)
     inner = pd.merge(left, right, how="outer", indicator=True, on=on)
-    return inner["_merge"].apply(lambda x: True if x == "both" else False).astype(bool)
+    return inner["_merge"].apply(
+        lambda x: True if x == "both" else False
```

Comment:
Would `inner["_merge"] == "both"` work here?

Reply:
That worked in my local test case that is included in the linked issue but failed CI with a different set of errors. Seems like progress at least.

Reply:
I think that change causes the length of the list to change (which caused the different errors) whereas mine just replaced everything with something. I'll try and find a more pandas-way to get the same result. I'm waiting for nix to finish installing on a linux box so I can hopefully run the tests locally.

Reply:
FWIW you can use nix, conda or pip to develop with ibis.